### DISCUSSION TITLE
Android sample: draw onto texture from TextureView in regular Activity

### DISCUSSION THREAD

**Discussion by netherpyro** [DiligentGraphics/DiligentEngine#185] [Q&A] [answered]
Hi! Thank you for the great lib. Could you please describe in general terms how can I change the Android sample to use TextureView inside Activity instead of NativeActivity to render on to?

**Comment by TheMostDiligent** [comment, marked as answer]
I suppose you need to extract a native handle from the view and then you can create a Diligent texture using either [IRenderDeviceGL::CreateTextureFromGLHandle](https://github.com/DiligentGraphics/DiligentCore/blob/master/Graphics/GraphicsEngineOpenGL/interface/RenderDeviceGL.h) or [IRenderDeviceVk::CreateTextureFromVulkanImage](https://github.com/DiligentGraphics/DiligentCore/blob/master/Graphics/GraphicsEngineVulkan/interface/RenderDeviceVk.h) depending on the API you are using.
